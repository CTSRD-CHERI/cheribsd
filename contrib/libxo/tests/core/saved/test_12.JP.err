test_12.test: invalid XML tag name: '2by4'
test_12.test: invalid XML tag name: '4x4'
test_12.test: invalid XML tag name: '2morrow'
test_12.test: invalid XML tag name: '2by4'
