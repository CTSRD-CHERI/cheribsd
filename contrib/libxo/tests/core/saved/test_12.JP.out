{
  "top": {
    "data": {
      "animal": "fish",
      "animal": "fish",
      "thing": [
        {
          "name": "thing",
          "color": "green",
          "time": "2:15",
          "hand": "left",
          "color": "blue",
          "time": "3:45"
        },
        {
          "name": "thing",
          "color": "green",
          "time": "2:15",
          "hand": "left",
          "color": "blue",
          "time": "3:45"
        },
        {
          "name": "thing",
          "color": "green",
          "time": "2:15",
          "hand": "left",
          "color": "blue",
          "time": "3:45"
        },
        {
          "name": "thing",
          "color": "green",
          "time": "2:15",
          "hand": "left",
          "color": "blue",
          "time": "3:45"
        },
        {
          "name": "thing",
          "color": "green",
          "time": "2:15",
          "hand": "left",
          "color": "blue",
          "time": "3:45"
        },
        {
          "name": "thing",
          "color": "green",
          "time": "2:15",
          "hand": "left",
          "color": "blue",
          "time": "3:45"
        },
        {
          "name": "thing",
          "color": "green",
          "time": "2:15",
          "hand": "left",
          "color": "blue",
          "time": "3:45"
        },
        {
          "name": "thing",
          "color": "green",
          "time": "2:15",
          "hand": "left",
          "color": "blue",
          "time": "3:45"
        },
        {
          "name": "thing",
          "color": "green",
          "time": "2:15",
          "hand": "left",
          "color": "blue",
          "time": "3:45"
        },
        {
          "name": "thing",
          "color": "green",
          "time": "2:15",
          "hand": "left",
          "color": "blue",
          "time": "3:45"
        }
      ]
    }
  }
}
